README.md

```diff
@@ -99,21 +99,20 @@ else
 end
 ```
 
-To conditionally activate plugins, `vim-plug` has a
-[few solutions](https://github.com/junegunn/vim-plug/wiki/tips#conditional-activation). For example, using the `Cond`
-helper, you can conditionally activate installed plugins
-([source](https://github.com/asvetliakov/vscode-neovim/issues/415#issuecomment-715533865)):
-
+Here is how to enable easy-motion using `vim-plug`:
 ```vim
 " inside plug#begin:
-" use normal easymotion when in VIM mode
-Plug 'easymotion/vim-easymotion', Cond(!exists('g:vscode'))
-" use VSCode easymotion when in VSCode mode
-Plug 'asvetliakov/vim-easymotion', Cond(exists('g:vscode'), { 'as': 'vsc-easymotion' })
+if exists('g:vscode')
+  " use VSCode easymotion when in VSCode mode
+  Plug 'asvetliakov/vim-easymotion', { 'as': 'vsc-easymotion' }
+else
+  " use normal easymotion when in VIM mode
+  Plug 'easymotion/vim-easymotion'
+endif
 ```
+For more information please visit `vim-plug` [wiki page](https://github.com/junegunn/vim-plug/wiki/tips#conditional-activation).
```

Comment:
There are zillions of plugin managers, we don't need to link to their docs. Users are expected to know how to use things they have chosen to install.

Reply:
Do you want me to remove this sentence?

Reply:
yeah, otherwise we might get more and more of this, people tend to mimic what's already there. the readme is too verbose already.